### ISSUE TITLE
Character Muse

### ISSUE THREAD

**Issue by barrysheppard** [nerdhaus/eldrich#38]
Under the Add New Muse part of the character would it be possible to copy the raw details from a default muse? That would save having to populate the stats and base skills every time.

Due to the number of useful skills that they have, it would be good to have these appear on the main character screen also.

**Comment by eaton** [comment]
Yes — that's currently done for Morphs (if you create a new morph instance, and ONLY select the morph type before saving, it will add all the default values from that morph to the instance itself). I'll add a similar behavior to Muses, then start taking a stab at making it more intuitive.
